lib/import_utils.js

```diff
@@ -2,13 +2,19 @@
 
 var fs = require("fs");
 var debug = require("./util/debug");
+var merge = require("lodash.merge");
 
-function ImportUtilities(eyeglass, sass, options, fallbackImporter) {
+function ImportUtilities(eyeglass, sass, options, fallbackImporter, context) {
   this.root = options.root;
   this.eyeglass = eyeglass;
   this.sass = sass;
   this.alreadyImported = {};
   this.fallbackImporter = fallbackImporter;
+  this.context = merge(context, {
+    eyeglass: {
+      imported: {} // keep track of files already imported @see importOnce
+    }
+  });
```

Comment:
Ah, nvm, I see from the merge documentation that because `imported` is an object this is basically a no-op.